src/api/generate-xlsx/styled-columns/is-title-row/index.ts

```diff
@@ -0,0 +1,24 @@
+import SECTION_NAMES from '../../../constants/XLSX-CONFIG/SECTION_NAMES';
+
+const { APPLICATION_INFORMATION } = SECTION_NAMES;
+
+/**
+ * isTitleRow
+ * Check if a row is a title row.
+ * @param {String} sheetName: ExcelJS worksheet name
+ * @param {Integer} rowNumber: Row number
+ * @returns {Boolean}
+ */
+const isTitleRow = (sheetName: string, rowNumber: number) => {
+  const isInfoSheet = sheetName === APPLICATION_INFORMATION;
+  const isInfoTitleOne = isInfoSheet && rowNumber === 8;
```

Comment:
would these numbers be better in a constants file?

Reply:
I had a play with this - have updated the `XLSX_ROW_INDEXES` constants to have a `APPLICATION_INFORMATION` object.

The only negative with this is that it is intentionally _not_ listed in the main index file for `XLSX_ROW_INDEXES`. I have updated the documentation to explain why.